openmetadata-ui/src/main/resources/ui/src/components/AddDataQualityTest/TestSuiteIngestion.tsx

```diff
@@ -103,13 +103,15 @@ const TestSuiteIngestion: React.FC<TestSuiteIngestionProps> = ({
   };
 
   const createIngestionPipeline = async (repeatFrequency: string) => {
+    const updatedName = testSuite.name.replace(/ /g, '_');
```

Comment:
Can we use the `replaceSpaceWith_` common util here?

Reply:
Thanks I'll check!